apps/shared/avifpng.c

```diff
@@ -347,6 +423,17 @@ avifBool avifPNGRead(const char * inputFilename,
         rowPointers[y] = &rgb.pixels[y * rgb.rowBytes];
     }
     png_read_image(png, rowPointers);
+    // gAMA chunk should be applied if it has an accompanying cHRM chunk, AND there is no overriding sRGB or iCCP chunk.
```

Comment:
There is a new cICP chunk. Should we also consider that? Please ask Luca Versari, who added cICP support to Chrome.

Reply:
I've heard of it but I did not include it because as far as I can tell it's only in the draft spec for now (at https://w3c.github.io/PNG-spec/ ) and it's not supported by libpng right now.